pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md

```diff
@@ -312,7 +312,7 @@ pod "capi-controller-manager-d4b9c7c4c-hkqfl" deleted
 
 ## Prepare the Dex Addon for Kubernetes v1.21.6
 
-The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as well as with v1.20.11.
+The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as with v1.20.11.
```

Comment:
I think the previous wording was better here, as after this change the Dex Addon works with both Kubernetes versions.   If you think this is too wordy, perhaps something like "works correctly with both Kubernetes 1.21.6 and 1.20.11."

```suggestion
The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as well as with v1.20.11.
```

Reply:
Yeah - I just did some trickeration because the linter doesn't like "as well as" ... I'll do your alternative here.